rcgen/src/csr.rs

```diff
@@ -23,15 +21,15 @@ impl PublicKeyData for PublicKey {
 	}
 }
 
-/// Data for a certificate signing request
-pub struct CertificateSigningRequest {
+/// Parameters for a certificate signing request
+pub struct CertificateSigningRequestParams {
```

Comment:
Tiny nit: would be nicer to extract the rename into a separate commit, IMO.

Reply:
Pulled into a separate commit.